prysm.sh

```diff
@@ -130,11 +129,11 @@ function verify() {
     fi
     checkSum="shasum -a 256"
     hash shasum 2>/dev/null || {
-	checkSum="sha256sum"
-    	hash sha256sum 2>/dev/null || {
-		echo >&2 "SHA checksum utility not available. Either install one (shasum or sha256sum) or run with PRYSM_ALLOW_UNVERIFIED_BINARIES=1."
-		exit 1
-    	}
+        checkSum="sha256sum"
+        hash sha256sum 2>/dev/null || {
+        echo >&2 "SHA checksum utility not available. Either install one (shasum or sha256sum) or run with PRYSM_ALLOW_UNVERIFIED_BINARIES=1."
+        exit 1
+        }
```

Comment:
Indentation is still different here. Please fix

Reply:
sorry for my carelessness